### ISSUE TITLE
Issue with rack 2 alpha/Rails 5

### ISSUE THREAD

**Issue by nateberkopec** [MiniProfiler/rack-mini-profiler#189]
```
2016-01-14 12:53:14 -0500: Rack app error: #<NoMethodError: undefined method `path=' for #<Rack::File:0x007ffb259c7318>
Did you mean?  Pathname>
/Users/nateberkopec/.gem/ruby/2.3.0/gems/rack-mini-profiler-0.9.8/lib/mini_profiler/profiler.rb:123:in `serve_html'
/Users/nateberkopec/.gem/ruby/2.3.0/gems/rack-mini-profiler-0.9.8/lib/mini_profiler/profiler.rb:178:in `call'
/Users/nateberkopec/.gem/ruby/2.3.0/bundler/gems/rails-7a756b90214e/railties/lib/rails/engine.rb:522:in `call'
/Users/nateberkopec/.gem/ruby/2.3.0/gems/rack-2.0.0.alpha/lib/rack/deflater.rb:35:in `call'
/Users/nateberkopec/.gem/ruby/2.3.0/gems/puma-2.15.3/lib/puma/server.rb:541:in `handle_request'
/Users/nateberkopec/.gem/ruby/2.3.0/gems/puma-2.15.3/lib/puma/server.rb:388:in `process_client'
/Users/nateberkopec/.gem/ruby/2.3.0/gems/puma-2.15.3/lib/puma/server.rb:270:in `block in run'
/Users/nateberkopec/.gem/ruby/2.3.0/gems/puma-2.15.3/lib/puma/thread_pool.rb:106:in `block in spawn_thread'
```

Appears to occur when the `includes.js` that RMP uses is requested.

**Comment by jrgifford** [comment]
Can confirm.

**Comment by SamSaffron** [comment]
ouch ... is this happening in latest from GitHub 
 
@dgynn recently added rack 2 support.

**Comment by Nekromancer** [comment]
@SamSaffron it works on latest from github

**Comment by seanlinsley** [comment]
Confirmed that the latest version from Github works fine with Rails 5.

**Comment by SamSaffron** [comment]
closing as I just released a version that fixes this.

**Comment by googya** [comment]
The bug seems still there!

**Comment by googya** [comment]
```shell
2017-02-27 11:29:32 +0800: Rack app error handling request { GET /mini-profiler-resources/includes.js }
#<NoMethodError: undefined method `path=' for #<Rack::File:0x007fae7d8c1058>
Did you mean?  Pathname>
/Users/leslie/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/rack-mini-profiler-0.9.3/lib/mini_profiler/profiler.rb:123:in `serve_html'
/Users/leslie/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/rack-mini-profiler-0.9.3/lib/mini_profiler/profiler.rb:175:in `call'
/Users/leslie/.rbenv/versions/2.3.1/gemsets/plato/gems/railties-5.0.0/lib/rails/engine.rb:522:in `call'
/Users/leslie/.rbenv/versions/2.3.1/gemsets/plato/gems/puma-3.6.2/lib/puma/configuration.rb:225:in `call'
/Users/leslie/.rbenv/versions/2.3.1/gemsets/plato/gems/puma-3.6.2/lib/puma/server.rb:578:in `handle_request'
/Users/leslie/.rbenv/versions/2.3.1/gemsets/plato/gems/puma-3.6.2/lib/puma/server.rb:415:in `process_client'
/Users/leslie/.rbenv/versions/2.3.1/gemsets/plato/gems/puma-3.6.2/lib/puma/server.rb:275:in `block in run'
/Users/leslie/.rbenv/versions/2.3.1/gemsets/plato/gems/puma-3.6.2/lib/puma/thread_pool.rb:116:in `block in spawn_thread'
2017-02-27 11:29:33 +0800: Rack app error handling request { GET /mini-profiler-resources/includes.js }
#<NoMethodError: undefined method `path=' for #<Rack::File:0x007fae6186b548>
Did you mean?  Pathname>

```

**Comment by dgynn** [comment]
@googya You are using v0.9.3. This was fixed in v0.9.9.

**Comment by googya** [comment]
@dgynn I see, I should use master branch, thank you
